### ISSUE TITLE
Unable to install

### ISSUE THREAD

**Issue by EricWiener** [netguru/sticky-parallax-header#83]
Thanks for making your work available.

I attempted to install through yarn and I'm getting the error:
```
Unable to resolve module `../../assets/data/cards` from `node_modules/react-native-sticky-parallax-header/src/predefinedComponents/AvatarHeader/AvatarHeader.js`: 

None of these files exist:
  * node_modules/react-native-sticky-parallax-header/src/assets/data/cards(.native|.ios.js|.native.js|.js|.ios.json|.native.json|.json|.ios.ts|.native.ts|.ts|.ios.tsx|.native.tsx|.tsx)
  * node_modules/react-native-sticky-parallax-header/src/assets/data/cards/index(.native|.ios.js|.native.js|.js|.ios.json|.native.json|.json|.ios.ts|.native.ts|.ts|.ios.tsx|.native.tsx|.tsx)

RCTFatal
__28-[RCTCxxBridge handleError:]_block_invoke
_dispatch_call_block_and_release
_dispatch_client_callout
_dispatch_main_queue_callback_4CF
__CFRUNLOOP_IS_SERVICING_THE_MAIN_DISPATCH_QUEUE__
__CFRunLoopRun
CFRunLoopRunSpecific
GSEventRunModal
UIApplicationMain
main
start
0x0
```

I checked the node_modules folder for react-native-sticky-parallax-header and it seems like the`assets` folder isn't there (although I see it is on the GitHub repo). I get this error when I try to do:

```
import React from 'react'
import StickyParallaxHeader from 'react-native-sticky-parallax-header'

const TestScreen = () => (
  <>
    <StickyParallaxHeader headerType="TabbedHeader" />
    {/* <StickyParallaxHeader headerType="AvatarHeader" /> */}
    {/* <StickyParallaxHeader headerType="DetailsHeader" /> */}
  </>
)

export default TestScreen
```

**Comment by mkalayci35** [comment]
+1

**Comment by stephanoparaskeva** [comment]
Did anyone find a solution for this?

**Comment by IdaszakDaniel** [comment]
Fixed in [version 0.2.1](https://github.com/netguru/sticky-parallax-header/releases/tag/v0.2.1)
